lumicks/pylake/force_calibration/power_spectrum.py

```diff
@@ -53,6 +53,27 @@ def downsampled_by(self, factor, reduce=np.mean):
 
         return ba
 
+    def _exclude_range(self, excluded_ranges):
+        """Exclude given frequency ranges from the power spectrum.
+
+        This function can be used to exclude noise peaks.
+
+        Parameters
+        ----------
+        exclude_ranges - `list` [`tuple` (float, float)]
```

Comment:
Shouldn't this be `List[Tuple[float, float]]` in PEP484 notation?

Reply:
Actually, considering that we're using the friendlier numpydoc elsewhere, shouldn't we stick to that? So:
`list of tuple of float`. I always think that's a bit easier for new users to read than PEP484.